pkg/helm/helm.go

```diff
@@ -147,6 +152,16 @@ func (h *Hydrator) getChartVersion(ctx context.Context) error {
 	} else {
 		return fmt.Errorf("failed to get version from output of `helm show chart`, stdout: %s", string(out))
 	}
+
+	// we need to clear the local helm cache after running `helm show chart`,
+	// otherwise we can get an OOM error on autopilot clusters later during
+	// the rendering step
+	if err := os.RemoveAll(helmCacheHome); err != nil {
+		// we don't necessarily need to exit on error here, as it is possible that the later rendering
+		// step could still succeed, so we just log the error and continue
+		klog.Infoln("failed to clear helm cache: %w", err)
```

Comment:
Did you intend to use `Infof` instead of `Infoln`?

Reply:
Yup I did, thank you for the catch! Updated